estate/views/estate_property_views.xml

```diff
@@ -127,7 +121,7 @@
         <field name="model">property</field>
         <field name="arch" type="xml">
             <search>
-                <filter string="Available" name="available" domain="[('state', '=', 'available')]"/>
+                <filter string="Available" name="available" domain="[('state', '==', ['new', 'offer_received'])]"/>
```

Comment:
When someone leaves you a comment, it doesn't mean that they have the 100% of reason

You can ask why some comments were made or why do you think the comment is not needed

If you check my comment, I said "this should be using equal operator"

I meant that it "should not", I made a mistake

But if you see the comment again I've sent a screenshot about the warning, and it says that **"we should be using the 'in' operator"**

![image](https://user-images.githubusercontent.com/47955967/236877703-e1e04fc6-318d-4004-8770-749cde3e82ba.png)

If you look at the code, it doesn't make sense for two reasons:

1. state only stores one value ('new' or 'offer_received', ...)
2. we are comparing a full list object with some specific values

There is no possible outcome so that state is equal to a list with some values:

`'new' == ['new', 'offer=received']  # this will never be equal`

```suggestion
                <filter string="Available" name="available" domain="[('state', 'in', ['new', 'offer_received'])]"/>
```

Reply:
@sebasdrk17 I have made the necessary changes to the code to use the 'in' operator as suggested, and I will make sure to double-check for similar issues in the future.